### ISSUE TITLE
regarding  data sets and model

### ISSUE THREAD

**Issue by nazeer2014** [mathDR/reading-text-in-the-wild#10]
I am trying  to run your  code

i try to run  for datasets and model as follows

:~/reading-text-in-the-wild/DICT2$ python extract_dictnet_weights.py
  File "extract_dictnet_weights.py", line 40
    model_file = 'LOCATION OF dictnet.mat from http://www.robots.ox.ac.uk/~vgg/research/text/#sec-models NIPS
                                                                                                            ^
SyntaxError: EOL while scanning string literal

how to fix this 
please

**Comment by mathDR** [comment]
@nazeer2014 Did you download the dictnet model from the website?

**Comment by nazeer2014** [comment]
I see a file charnat.mat in  model_release directory  in 'reading-text-in-the-wild ' dowloaded directory

But i didnt download  dictnet model
(I think you talk about downloading from http://www.robots.ox.ac.uk/~vgg/research/text/model_release.tar.gz)

As i am new to this field could you please expalin  the in detail steps to run the madles and dataset in ubuntu platform
 
I hope it willbe helpful for new comers

Many thanks in advance

**Comment by mathDR** [comment]
Hi @nazeer2014 I was hoping the README file went through this and served as a primer on how to get started.  Can you point to instances where it is confusing?

**Comment by nazeer2014** [comment]
yes i went through README file

where i get confused is in dataset and model section:

""The training data for the networks comes from the MJSynth dataset and the models are extracted from the MATLAB models located at models

The weights from the MATLAB models are extracted for conversion to Keras via the files

extract_dictnet_weights.py

for the DICT+2 model ""

so when i run  extract_dictnet_weights.py

ubuntu says 

File "extract_dictnet_weights.py", line 40
model_file = 'LOCATION OF dictnet.mat from http://www.robots.ox.ac.uk/~vgg/research/text/#sec-models NIPS
^
SyntaxError: EOL while scanning string literal

so which file i should modify?

**Comment by mathDR** [comment]
The file you need is located at the address (robots.ox.ac.uk site).  The file is too big to host on github, so you need to download it yourself.  After downloading, you need to go into the extract_dictnet_weights.py file and overwrite the text of model_file to where *on your local machine* you placed the file.

**Comment by nazeer2014** [comment]
ah OK  I am downlading it  i will try

and it is same procedure for extract_charnet_weights.py also ?

**Comment by mathDR** [comment]
Yes,  the procedure is the same.  Furthermore, within the README, it denotes the correct versions of keras and tensorflow.  You will have to change some tensorflow files as well

**Comment by nazeer2014** [comment]
Many Thanks to you

I will check and come back to you if I need any help
